internal/volumes/mount.go

```diff
@@ -134,7 +139,19 @@ func (s *LinuxMountService) Publish(targetPath string, devicePath string, opts M
 		return s.mounter.Mount(devicePath, targetPath, opts.FSType, mountOptions)
 	}
 
-	return s.mounter.FormatAndMount(devicePath, targetPath, opts.FSType, mountOptions)
+	formatOptions := make([]string, 0)
+
+	if opts.FsFormatOptions != "" {
+		lexer := shell.NewLex('\\')
+		formatOptions, err = lexer.ProcessWords(opts.FsFormatOptions, shell.EnvsFromSlice([]string{}))
```

Comment:
I would personally find passing list of strings simpler than importing a shell lexer here, not a hard requirement

Reply:
Unfortunately we can´t do this as StorageClass parameters are unmarshalled into map[string]string.